.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,62 @@
+name: Bug report
+description: Report build and runtime bugs
+labels: ["Type: Bug"]
+body:
+  - type: checkboxes
+    id: checklist
+    attributes:
+      label: Answers checklist.
+      description: Before submitting a new issue, please follow the checklist and try to find the answer.
+      options:
+        - label: I have read the component documentation [ESP-IDF Components](https://components.espressif.com) and the issue is not addressed there.
+          required: true
+        - label: I am using target and esp-idf version as defined in component's idf_component.yml
+          required: true
+        - label: I have searched the [issue tracker](https://github.com/espressif/esp-usb/issues?q=is%3Aissue) for a similar issue and not found any related issue.
+          required: true
+  - type: dropdown
+    id: component
+    attributes:
+      label: Which component are you using? If you choose Other, provide details in More Information.
+      multiple: false
+      options:
+        - device/esp_tinyusb
+        - host/cdc
+        - host/hid
+        - host/msc
+        - host/uvc
+        - Other
```

Comment:
Should this be in esp-usb repository?

Reply:
I removed that part. 
Thanks for checking it.